### ISSUE TITLE
Ability to specify user

### ISSUE THREAD

**Issue by vtvz** [yii2tech/crontab#16]
Crontab allows me to specify which user I change schedule using param -u:

```
usage:  crontab [-u user] file
        crontab [ -u user ] [ -i ] { -e | -l | -r }
                (default operation is replace, per 1003.2)
        -e      (edit user's crontab)
        -l      (list user's crontab)
        -r      (delete user's crontab)
        -i      (prompt before deleting user's crontab)
``` 

But I cannot just set `binPath` as `crontab -u username` because it creates invalid constructions and then throw exception when I try to change crontab config

I wish you add ability to specify user

**Comment by klimov-paul** [comment]
Resolved by commit dfb553c5beb85649a912987d187a6e35ad8c3abf

**Comment by vtvz** [comment]
Thanks a lot)
